node/core/dispute-coordinator/src/lib.rs

```diff
@@ -316,6 +316,8 @@ where
 		rolling_session_window: RollingSessionWindow::new(DISPUTE_WINDOW),
 	};
 
+	resume(ctx, &mut state, backend).await?;
```

Comment:
it should probably be called in `run` outside of the loop

unrelated: the naming of this function `run_iteration` is a bit misleading, it should probably be called `run_until_error` or something like that